volta/meta.yaml

```diff
@@ -3,7 +3,7 @@
 
 package:
   name: {{ name }}
-  version: {{ version }}.memfault2
```

Comment:
I'm keeping open a thread here until I run the GHA and push the new builds to Conda